core/store/src/test/java/uk/gov/gchq/gaffer/store/StoreTest.java

```diff
@@ -246,6 +246,24 @@ public void after() {
         JSONSerialiser.update();
     }
 
+    @Test
+    public void shouldExecuteOperationWhenJobTrackerCacheIsBroken(@Mock final StoreProperties storeProperties) throws Exception {
```

Comment:
This test doesn't work because `StoreImpl` overrides the `createJobTracker()` method such that it doesn't create a JobTracker at all. Also, JobTracker is disabled by default and needs to be enabled but this can't be done because mocked store properties are used.

Reply:
Added a storeimpl with an enabled JobTracker which isnt mocked.